src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -101,6 +101,10 @@ def create(cls, catalog: CatalogName) -> 'RepositoryPlugin':
     def __init__(self, source: TDRSource) -> None:
         super().__init__()
         self._source = source
+        project = self.tdr.project_for_source(source)
+        require(source.project == project,
+                f'Source project is set to {source.project!r} but should be '
+                f'{project!r}')
```

Comment:
I wouldn't assume which one is authoritative. And we don't need elaborate interpolations for this type of thing. They just make for noisy syntax.

```suggestion
        require(project == source.project, 
                "Actual Google project of TDR source differs from configured one", 
                project, source)
```